src/Compute/Compute/Manual/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingExtensionStartUpgrade.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Compute.Automation.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.Compute.Automation
+{
+    [Cmdlet(VerbsLifecycle.Start, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VmssRollingExtensionUpgrade", DefaultParameterSetName = "DefaultParameter", SupportsShouldProcess = true)]
+    [OutputType(typeof(PSOperationStatusResponse))]
+    public partial class VirtualMachineScaleSetRollingExtensionStartUpgrade : ComputeAutomationBaseCmdlet
+    {
+        private const string ByResourceIdParamSet = "ByResourceId",
+            ByInputObjectParamSet = "ByInputObject", DefaultParameterSetName = "DefaultParameter";
```

Comment:
DefaultParameterSetName = "DefaultParameter" defined in line 9 also

Reply:
Set the parameter set for the DefaultParameter set parameters to the 'DefaultParameter' string.